neuraxle/steps/loop.py

```diff
@@ -146,6 +161,36 @@ def should_resume(self, data_container: DataContainer, context: ExecutionContext
         return False
 
 
+class ContinueException(Exception):
+    pass
+
+
+class BreakException(Exception):
+    pass
+
+
+class Break(Identity):
+    def __init__(self, condition_function: Callable):
+        Identity.__init__(self)
+        self.condition_function = condition_function
+
+    def _did_process(self, data_container: DataContainer, context: ExecutionContext) -> DataContainer:
+        if self.condition_function(data_container):
```

Comment:
Should we pass the context as well?

Reply:
Can we think of use-cases where the context would be required in determining if we need to interrupt? If so, we could add it as a param or optional param. Otherwise, that's already okay.